javascript/lang/en_GB.js

```diff
@@ -13,6 +13,7 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "UserForms.ADDING_OPTION": "Adding option",
     "UserForms.ADDING_RULE": "Adding rule",
     "UserForms.CONFIRM_DELETE_ALL_SUBMISSIONS": "All submissions will be permanently removed. Continue?",
+	"UserForms.ERROR_CONTAINER_HEADER": "Please correct the following errors and try again:",
```

Comment:
Hi @sebastiand - sorry that nobody has replied to you sooner. If you would still like this to be merged, could you please only update `javascript/lang/en.js` and `javascript/lang/src/en.js`? The other files will be updated automatically by Transifex when it's next updated.